modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java

```diff
@@ -438,6 +444,18 @@ public OperationsMap postProcessOperationsWithModels(OperationsMap objs, List<Mo
         return objs;
     }
 
+    private boolean isApplicationJsonOrApplicationXml(CodegenOperation op) {
+       if (op.produces != null) {
+           for(Map<String, String> produce : op.produces) {
+              if ("application/json".equalsIgnoreCase(produce.get("mediaType"))
```

Comment:
what about using `isJsonMimeType`, `isXmlMimeType` in default codegen to check if the mime type is json/xml related?

Reply:
That makes sense (I was not aware of those methods).
Thanks a lot for this pointer!